lib/vrl/stdlib/src/log.rs

```diff
@@ -21,6 +21,11 @@ impl Function for Log {
                 kind: kind::BYTES,
                 required: false,
             },
+            Parameter {
+                keyword: "rate_limit_secs",
+                kind: kind::INTEGER,
```

Comment:
I'm a bit late on the review, but maybe one follow-up here: I can imagine users wanting to specify rate limits such as `0.1s` or `1.5s`.

Does `internal_log_rate_secs` not support floats, or is there any other reason why fractions of a second shouldn't be allowed?

Reply:
The internal rate limiting logic only handles seconds right now. Interestingly it looks like tracing doesn't have support for passing floats around (https://docs.rs/tracing/0.1.25/tracing/trait.Value.html) so we'd need to adjust it to be something like `internal_rate_limit_ms` to allow finer granularity. Here it could still be represented as a float though.

Reply:
Opened https://github.com/timberio/vector/issues/6643 to track this one.